PetGame/Controllers/LeaderboardController.cs

```diff
@@ -5,85 +5,63 @@
 
 namespace PetGame
 {
+    public class LeaderboardEntry
+    {
+        public string PetName { get; set; }
+        public int score { get; set; }
+        public string OwnerName { get; set; }
+
+        public LeaderboardEntry(string nPetName, int nScore, string nOwnerName)
```

Comment:
This constructor is unnecessary, initialization can be done with the following:
```cs
var x = new LeaderboardEntry() { PetName = "aaa", Score = 44, OwnerName = "person" };
```

Also, why wouldn't this type include all of the fields from Race, including the time when the score was set and the various IDs?